src/AbstractFixer.php

```diff
@@ -80,6 +86,63 @@ public function supports(\SplFileInfo $file)
         return true;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration) {
+            @trigger_error(
+                'Passing NULL to set default configuration is deprecated and will not be supported in 3.0, use an empty array instead.',
+                E_USER_DEPRECATED
+            );
```

Comment:
If we want to have a deprecation let us make it the way it will follow separation of concerns. If not, then no need for introducing deprecation.